dataland-frontend/src/components/pages/TermsAndConditions.vue

```diff
@@ -1,587 +1,1096 @@
 <template>
   <LegalPage>
-    <h1>Allgemeine Bedingungen für die Teilnahme an Dataland</h1>
-    <h1>(„AGB-Dataland“)</h1>
-    <div>
-      <h2>Vorbemerkung</h2>
-      <p>
-        Die Dataland GmbH (nachfolgend: „Dataland“) betreibt eine Plattform (nachfolgend: „Plattform-Dataland“, um
-        unternehmensbezogene Daten, insbesondere in den Bereichen Umwelt („Environmental“), Soziales („Social“) und
-        Unternehmensführung („Governance“) zusammenzutragen und für ihre Nutzer (nachfolgend: „Daten-User“) verfügbar zu
-        machen (nachfolgend „Unternehmens-Daten“).
-      </p>
-      <p>
-        Die auf der Plattform-Dataland registrierten (natürlichen und juristischen) Personen (nachfolgend:
-        „Dataland-Teilnehmer“) können als sogenannte Daten-Sammler Unternehmens-Daten an Dataland übertragen. Das
-        Gleiche können Unternehmen mit den Unternehmens-Daten tun, die sie selbst betreffen (sog. „Daten-Owner“). Die
-        Unternehmens-Daten auf der Plattform-Dataland können von den Daten-Usern heruntergeladen und genutzt werden.
-      </p>
-      <p>
-        Ziel von Dataland ist es, eine allen offenstehende Wissensplattform zu schaffen, und die Unternehmens-Daten
-        allen Daten-Usern zur Nutzung zur Verfügung zu stellen. Dataland prüft die eingegebenen Unternehmens-Daten mit
-        automatisierten Tests. Darüber hinaus werden Stichproben mit Einzeldaten durchgeführt. Ansonsten erfolgt keine
-        inhaltliche Überprüfung der von den Daten-Sammlern oder den Daten-Ownern an die Plattform-Dataland übermittelten
-        und dort zur Verfügung gestellten Unternehmens-Daten.
-      </p>
-      <p>
-        Die folgenden Bedingungen regeln die allgemeine Registrierung als Dataland-Teilnehmer sowie die Registrierung
-        und Teilnahme an Dataland als Daten-User, als Daten-Sammler und als Daten-Owner.
-      </p>
+    <div class="flex justify-content-start align-items-center">
+      <img src="/public/static/icons/uk_flag.jpg" height="20" alt="English" />
+      <InputSwitch
+        data-test="terms-language-toggle-button"
+        v-model="languageGerman"
+        class="p-inputswitch-singlecolor"
+        style="margin-left: 10px; margin-right: 10px"
+      />
+      <img src="/public/static/icons/germany_flag.png" height="20" alt="German" />
```

Comment:
This component is not visible in the mobile version of dataland.com That needs to be fixed.

![image](https://github.com/d-fine/Dataland/assets/35459207/9ff6e9ed-dd5e-4277-bf37-468e2bc7cd15)

Reply:
Adjusted the margins to make the language switch stay below the header, checked it on desktop and mobile window for all screen widths

Reply:
Adjusted the margins to make the language switch stay below the header when not signed in (as that makes the header taller), checked it on desktop and mobile window for all screen widths.
Below 290px width, the header starts looking weird and the language switch is no longer visible but I think its ok:
![image](https://github.com/d-fine/Dataland/assets/165936434/2e7b6739-c6c4-4e63-8b36-61cf94def905)